litmus-portal/graphql-server/graph/schema.resolvers.go

```diff
@@ -63,22 +66,52 @@ func (r *mutationResolver) DeleteChaosWorkflow(ctx context.Context, workflowid s
 }
 
 func (r *mutationResolver) SendInvitation(ctx context.Context, member model.MemberInput) (*model.Member, error) {
+	err := validate.ValidateRole(ctx, member.ProjectID, requiredRoles[2:3], []string{"Accepted"})
```

Comment:
Refactor the calls to include the roles directly as an array instead of doing it as a slice of some predefined list. This might become unmaintainable later when we have different combination of roles required.

```suggestion
	err := validate.ValidateRole(ctx, member.ProjectID, [].models.MemberRole{MemberRoleOwner}, []string{"Accepted"})
```